src/oom/oomd-manager.c

```diff
@@ -127,6 +127,12 @@ static int process_managed_oom_message(Manager *m, uid_t uid, JsonVariant *param
                         ctx->mem_pressure_limit = limit;
         }
 
+        /* Toggle wake-ups for "ManagedOOMSwap" if entries are present. */
+        r = sd_event_source_set_enabled(m->swap_context_event_source,
```

Comment:
Hmm, a failure to disable the source (i.e., set `SD_EVENT_OFF`) is benign, as it just results in some spurious wakeups, but a failure to enable it (`SD_EVENT_ON`) means this doesn't work at all. Should that really be ignored?

Reply:
I guess the question is what would you want to do in that case?

Assuming this is a memory pressure source, it looks like the failure case consists of `epoll_ctl()` failing?

Reply:
Good catch. Failure to enable the event should not pass under radar (the debug level message).
While it's unlikely that `sd_event_source_set_enabled(, SD_EVENT_ON)`, it's a good approach to propagate that error upwards.